tests/test_jointcal_cfht.py

```diff
@@ -55,7 +55,8 @@ def test_jointcalTask_2_visits(self):
         # first run of jointcal on this data. We should always do better than
         # this in the future!
         relative_error = 25e-3*u.arcsecond
-        self._testJointCalTask(2, relative_error, absolute_error)
+        pa1 = 0.019
```

Comment:
I've expanded the text in that docstring a bit to clarify, and added Tim's link (I didn't know there was a shortened version).